code/espurna/sensors/T6613Sensor.h

```diff
@@ -0,0 +1,186 @@
+// -----------------------------------------------------------------------------
+// T6613 CO2 sensor
+// https://www.amphenol-sensors.com/en/telaire/co2/525-co2-sensor-modules/321-t6613
+// Uses SoftwareSerial library
+// -----------------------------------------------------------------------------
+
+#if SENSOR_SUPPORT && T6613_SUPPORT
+
+#pragma once
+
+#include "Arduino.h"
+#include "BaseSensor.h"
+#include <SoftwareSerial.h>
+
+#define T6613_REQUEST_LEN       5
+#define T6613_RESPONSE_LEN      5
+#define T6613_TIMEOUT           1000
+#define T6613_GETPPM            0x020203
+
+class T6613Sensor : public BaseSensor {
+
+    public:
+
+        // ---------------------------------------------------------------------
+        // Public
+        // ---------------------------------------------------------------------
+
+        T6613Sensor(): BaseSensor() {
+            _count = 1;
+            _sensor_id = SENSOR_T6613_ID;
+        }
+
+        ~T6613Sensor() {
+            if (_serial) delete _serial;
+        }
+
+        // ---------------------------------------------------------------------
+
+        void setRX(unsigned char pin_rx) {
+            if (_pin_rx == pin_rx) return;
+            _pin_rx = pin_rx;
+            _dirty = true;
+        }
+
+        void setTX(unsigned char pin_tx) {
+            if (_pin_tx == pin_tx) return;
+            _pin_tx = pin_tx;
+            _dirty = true;
+        }
+
+        // ---------------------------------------------------------------------
+
+        unsigned char getRX() {
+            return _pin_rx;
+        }
+
+        unsigned char getTX() {
+            return _pin_tx;
+        }
+
+        // ---------------------------------------------------------------------
+        // Sensor API
+        // ---------------------------------------------------------------------
+
+        // Initialization method, must be idempotent
+        void begin() {
+
+            if (!_dirty) return;
+
+            if (_serial) delete _serial;
+
+            _serial = new SoftwareSerial(_pin_rx, _pin_tx, false, 32);
+            _serial->enableIntTx(false);
+            _serial->begin(19200);
```

Comment:
Add as const / define above?

Reply:
For this sensor, the baud rate is fixed to 19200 and it was also an inline integer in MHZ19.